.github/actions/zango-project-setup-action/action.yml

```diff
@@ -0,0 +1,93 @@
+name: Zango project setup with latest zango build
+
+on:
+  workflow_call:
+    inputs:
```

Comment:
@DevilsAutumn Can we make this inputs optional and add default values ?

Reply:
inputs with defaults are failing, see: https://github.com/Healthlane-Technologies/Zango/actions/runs/10198463859

Reply:
postgres container throwing error on setting default values and not passing from `e2e-cypress-tests.yml`.